prin

```diff
@@ -142,15 +144,15 @@ if [[ ${POSITIONAL_ARGV[0]} =~ [0-9]{5,} ]]; then
   # non-cherrypick commit searches on branches
   INITIAL_COMMIT=$(git log --grep "Merge pull request #$PR" \
                            --all --pretty=format:"%h")
-  CP_COMMITS=($(git log --grep "cherry-pick-of-.*#$PR-" \
-                        --all --pretty=format:"%h"))
+  mapfile -i CP_COMMITS < <(git log --grep "cherry-pick-of-.*#$PR-" \
+                        --all --pretty=format:"%h")
 else
   # TODO: Need to validate this is a git hash
   INITIAL_COMMIT=("${POSITIONAL_ARGV[0]}")
 fi
 
-[[ -z "$INITIAL_COMMIT" ]] && common::exit 1 "No commit/pr found in this repo."
+[[ -z "${INITIAL_COMMIT[0]}" ]] && common::exit 1 "No commit/pr found in this repo."
 
 security_layer::hosted_map
 
-show_tags "$PR" $INITIAL_COMMIT ${CP_COMMITS[*]}
+show_tags "$PR" "${INITIAL_COMMIT[0]}" "${CP_COMMITS[*]}"
```

Comment:
There is a good chance this should be `"${CP_COMMITS[@]}"` instead, ref: https://github.com/koalaman/shellcheck/wiki/SC2086#exceptions

Reply:
Thanks a lot for reading these. I'm sure there is a lot of hidden bugs/weirdness from my shellcheck changes. I will create an issue and link this, and other things found, to it